### DISCUSSION TITLE
max-... (like max-lg: or max-xl:) classes doesn't work anymore

### DISCUSSION THREAD

**Discussion by AlexisATI4** [tailwindlabs/tailwindcss#13078] [Help]
Hello, I often use max- classes... but they don't seem to be compiled anymore. For example, if I do `max-lg:flex`, it doesn't work anymore. I'm on Magento 2 with the Hyva theme, which is currently running Tailwind 3.2.7

**Comment by wongjn** [comment]
Consider checking if you have customized your `screens` configuration with any non-"simple" values, see #9558, the _Restrictions_ section.

Otherwise, consider checking if your code works in [Tailwind Play](https://play.tailwindcss.com). If it does, then it could be some misconfiguration/misuse. In which case, consider providing a project that reproduces the unexpected behavior.
